modules/340-extended-monitoring/images/extended-monitoring-exporter/src/extended-monitoring.go

```diff
@@ -0,0 +1,232 @@
+package main
+
+import (
```

Comment:
You also need to correct the helm template for the golang variant  https://github.com/Horiodino/deckhouse/blob/extended-monitoring-alerts-separately-%231311/modules/340-extended- monitoring/templates/extended-monitoring-exporter/deployment.yaml#L64-L86